rules/windows/process_creation/win_susp_use_of_sqlps_bin.yml

```diff
@@ -0,0 +1,28 @@
+title: Detection of PowerShell Execution via SQL 
+id: 0152550d-3a26-4efd-9f0e-54a0b28ae2f3  
+status: experimental
+description: PowerShell execution through builtin SQL Server "SQLPS.exe" binary. Microsoft PS logging like
+  ScriptBlock logging function of PowerShell is not an option here, PS session caused by the binary won't be recorded/logged.
+references:
+  - https://twitter.com/pabraeken/status/993298228840992768
```

Comment:
Sorry, the tweet link in your rule is to the wrong tweet. That tweet to the neighboring utility - `sqltoolsps.exe`. 

Correct link: https://twitter.com/bryon_/status/975835709587075072